### ISSUE TITLE
messy TOC

### ISSUE THREAD

**Issue by ghost** [nevir/groc#11]
In the TOC folders are mixed with markdown headers.

This makes it difficult to orientate.

**Comment by nevir** [comment]
Agree.  I think the combined approach is too difficult to have distinct styles from, and would benefit from additional hierarchy.

What about this:
- The ToC is split into two hierarchies - one for the file/directory structure, the other being the headers for the _current file_
- Search still searches all files & headers

**Comment by ghost** [comment]
I think that would make it simple to understand!
